appengine-java11/custom-entrypoint/Main.java

```diff
@@ -23,8 +23,10 @@
 public class Main {
 
   public static void main(String[] args) throws IOException {
-    // Create an instance of HttpServer bound to port 8080.
-    HttpServer server = HttpServer.create(new InetSocketAddress(8080), 0);
+    // Create an instance of HttpServer bound to port defined by the 
+    // PORT environment variable when is present, otherwise on 8080.
```

Comment:
Sorry I wasn't more clear - can you update this in the other files as well? 
```suggestion
    // PORT environment variable when present, otherwise on 8080.
```

Reply:
You right!
This is my first time here, and still reading guidelines...
Excuse me for be slowly.
;-)